pysisyphus/calculators/PySCF.py

```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 import pyscf
-from pyscf import gto, grad, lib, hessian, tddft, qmmm
```

Comment:
Weren't these imports necessary as they had some kind of side effects?! Did this change in recent PySCF versions?

Reply:
Those are imports not used in this file, I'm just following my IDE. And this is reasonable, the removed packages are encapsulated into the mf object, for example when calculating the hessian, mf.Hessian() is called, rather than we touching the hessian package directly.

Reply:
I changed to `from pyscf import __all__` . This ensures all submodules are loaded in the `pyscf` namespace